kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -69,7 +73,28 @@
     margin: 5px;
     position: relative;
     top: -3px;
+    letter-spacing: .5pt;
+}
+
+.kb-data-staging-button:hover{
```

Comment:
you could remove all the `border-radius` directives from the class states and assign them directly to the class, i.e. 
```css
.kb-data-staging-button {
  border-radius: 4px;
}
```

Reply:
good idea